homeassistant/components/rfxtrx.py

```diff
@@ -0,0 +1,95 @@
+"""
+homeassistant.components.rfxtrx
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Connects Home Assistant to a RFXtrx device.
+
+Configuration:
+
+To use Rfxtrx device you will need to add the following to your
+configuration.yaml file.
+
+rfxtrx:
+  device: /dev/serial/by-id/usb-RFXCOM_RFXtrx433_A1YVC1P0-if00-port0
+
+*Optional*
+
+  debug: True
+
+"""
+import logging
+from homeassistant.util import slugify
+
+DEPENDENCIES = []
+REQUIREMENTS = ['https://github.com/Danielhiversen/pyRFXtrx/archive/' +
+                'ec7a1aaddf8270db6e5da1c13d58c1547effd7cf.zip#RFXtrx==0.15']
+
+DOMAIN = "rfxtrx"
+CONF_DEVICE = 'device'
+CONF_DEBUG = 'debug'
+RECEIVED_EVT_SUBSCRIBERS = []
+RFX_DEVICES = {}
+_LOGGER = logging.getLogger(__name__)
+RFXOBJECT = None
+
+
+def setup(hass, config):
+    """ Setup the Rfxtrx component. """
+
+    # Declare the Handle event
+    def handle_receive(event):
+        """ Callback all subscribers for RFXtrx gateway. """
+
+        # Log RFXCOM event
+        entity_id = slugify(event.device.id_string.lower())
+        packet_id = "".join("{0:02x}".format(x) for x in event.data)
+        entity_name = "%s : %s" % (entity_id, packet_id)
+        _LOGGER.info("Receive RFXCOM event from %s => %s",
+                     event.device, entity_name)
+
+        # Callback to HA registered components
+        for subscriber in RECEIVED_EVT_SUBSCRIBERS:
+            subscriber(event)
+
+    # Try to load the RFXtrx module
+    try:
+        import RFXtrx as rfxtrxmod
+    except ImportError:
+        _LOGGER.exception("Failed to import rfxtrx")
+        return False
+
+    # Init the rfxtrx module
+    global RFXOBJECT
+
+    device = config[DOMAIN][CONF_DEVICE]
+    try:
```

Comment:
Dicts support the `get` method that takes in a second argument that will be returned if key does not exists.
```python
debug = config[DOMAIN].get(CONF_DEBUG, False)
```